docs/source/models/supported_models.rst

```diff
@@ -168,3 +168,49 @@ Alternatively, you can raise an issue on our `GitHub <https://github.com/vllm-pr
         llm = LLM(model=..., revision=..., trust_remote_code=True)  # Name or path of your model
         output = llm.generate("Hello, my name is")
         print(output)
+
+Model Support Policy
+---------------------
+
+At vLLM, we are committed to facilitating the integration and support of third-party models within our ecosystem. Our approach is designed to balance the need for robustness and the practical limitations of supporting a wide range of models. Here’s how we manage third-party model support:
+
+1. **Community-Driven Support**: We encourage community contributions for adding new models. When a user requests support for a new model, we welcome pull requests (PRs) from the community. These contributions are evaluated primarily on the sensibility of the output they generate, rather than strict consistency with existing implementations such as those in transformers.
```

Comment:
I think we should call out in particular that model support PRs directly from the model vendors are greatly appreciated.

Reply:
fixed in [258d1bb](https://github.com/vllm-project/vllm/pull/3948/commits/258d1bb9910e9e48b079a758f3b0f5b879aa00eb)